huggingface/pytorch/buildspec.yml

```diff
@@ -42,3 +42,31 @@ images:
                  *CUDA_VERSION, '-', *OS_VERSION ]
     docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /,
                          *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+  BuildHuggingFacePytorchCpuPy36InferenceDockerImage:
```

Comment:
If you are trying to build cu110, the CUDA_VERSION arg on line 37 should be changed

Reply:
Why? It seems that the Training DLC uses `cu111` and Inference DLC `cu110`. Is it required to have the same across all?